src/s2/util/math/exactfloat/exactfloat_test.cc

```diff
@@ -119,14 +120,14 @@ double scalbln(double a, long exp) {
 // Here we fix the rounding functions to match MPFloat, which clamps out of
 // range values and returns the maximum possible value for NaN.
 
-#define FIX_INT_ROUNDING(T, fname)                      \
```

Comment:
Is something changing? Don't reformat unrelated code, it makes it hard to see what's actually going on.